docs/Writing_yeb_easyconfig_files.rst

```diff
@@ -117,6 +117,37 @@ elsewhere too) can be done as follows::
     versionsuffix: !join [-Python-, *pyver]
 
 
+.. _easyconfig_yeb_format_syntax_escaping:
+
+Escaping string values with single or double quotes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Strings in YAML can be unquoted. However, when they contain special characters they need to be escaped by either single-
+or double-quoting the string.
+
+Special characters that require single quotes are: `:` `{` `}` `[` `]` `,` `&` `*` `#` `?` `|` `-` `<` `>` `=` `!` `%` `@` and `\``.
+When using single-quoted strings, any single quote inside the string must be doubled to escape it.
+
+If the string contains control characters such as `\n`, it must be escaped with double quotes. 
+
+
+.. _easyconfig_yeb_format_syntax_version_majorminor:
+
+Extract major.minor from the version string
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@Caylo please remove this part since we closed https://github.com/hpcugent/easybuild-framework/pull/1444 ?